pulp_rpm/app/models/repository.py

```diff
@@ -51,6 +59,10 @@ class RpmRepository(Repository):
     ]
 
     sub_repo = models.BooleanField(default=False)
+    revision_number = models.IntegerField(null=True)
+    remote_id = models.CharField(max_length=255, null=True)
```

Comment:
This should be a foreign key to an RpmRemote. Similar to how Distributions have a relation to it.

https://github.com/pulp/pulpcore/blob/master/pulpcore/app/models/publication.py#L290